_config.yml

```diff
@@ -9,6 +9,7 @@
 title: Edit _config.yml to set your title!
 description: This is where the description of your site will go. You should change it by editing the _config.yml file. It can be as long as you like! Happy blogging... ❤
 github_username: jph00
+url: # the full url for your site for your site, e.g. http://username.github.io/blog
```

Comment:
This is optional, at least in some cases, right? If so, move it after email and twitter, and add a comment above saying it's optional, and explain when it's needed.

Might it be possible to use the metadata from this plugin instead?: https://github.com/jekyll/github-metadata . I've used it for the social icons already.